packages/react-components/src/components/ColorFacet/styles.css

```diff
@@ -33,6 +33,11 @@
   }
 }
 
+.ballMobile {
```

Comment:
Better to use .ball-mobile, both versions will work, but compiled version of camelcase will be not converted to a dash `.findify-components...-ballMobile`